src/server/pps/iface.go

```diff
@@ -18,5 +20,7 @@ type APIServer interface {
 	StopJobInTransaction(*txncontext.TransactionContext, *pps_client.StopJobRequest) error
 	UpdateJobStateInTransaction(*txncontext.TransactionContext, *pps_client.UpdateJobStateRequest) error
 	CreatePipelineInTransaction(*txncontext.TransactionContext, *pps_client.CreatePipelineRequest, *string, *uint64) error
-	InspectPipelineInTransaction(*txncontext.TransactionContext, string, bool) (*pps_client.PipelineInfo, error)
+	InspectPipelineInTransaction(*txncontext.TransactionContext, string) (*pps_client.PipelineInfo, error)
+
+	ListPipelineCallback(context.Context, *pps_client.ListPipelineRequest, func(*pps_client.PipelineInfo) error) error
```

Comment:
I think we use `ForEachX` for this elsewhere.  It's internal, so we can change it later.